runtime/doc/os_win32.txt

```diff
@@ -7,8 +7,8 @@
 						*win32* *Win32* *MS-Windows*
 This file documents the idiosyncrasies of the Win32 version of Vim.
 
-The Win32 version of Vim works on Windows NT, 95, 98, ME, XP, Vista and
-Windows 7.  There are both console and GUI versions.
+The Win32 version of Vim works on Windows NT, XP, Vista and Windows 7.
```

Comment:
Probably, but I don't want to do it right now since Windows support isn't even merged in yet.